custom_components/onewire_sysbus/translations/sk.json

```diff
@@ -0,0 +1,18 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Zariadenie je už nakonfigurované"
+        },
+        "error": {
+            "invalid_path": "Adresár nenájdený."
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "mount_dir": "Pripojiť adresár"
+                },
+                "title": "Nastaviť 1-Wire"
```

Comment:
"Nastaviť" - I didn't find a translation for this word, can you explain it or show a similar reference in another intergration?

Reply:
Set up (EN) = Nastaviť (SK)
used too direct in Home Assistant translation

![image](https://github.com/thecode/ha-onewire-sysbus/assets/61636045/15a73b8a-dbe8-4c8f-bbf8-616c6caaa6a3)